1_fetch/src/calc_HIT.R

```diff
@@ -2,7 +2,7 @@
 
 
 calc_HITmetrics<-function(data,yearType,drainArea,floodThreshold){
-  
+  suppressWarnings(calc_allHIT(x=data,yearType=yearType,drainArea=drainArea,floodThreshold=floodThreshold))
   out_data<-calc_allHIT(x=data,yearType=yearType,drainArea=drainArea,floodThreshold=floodThreshold)
```

Comment:
This should be `out_data<-suppressWarnings(calc_allHIT(x=data,yearType=yearType,drainArea=drainArea,floodThreshold=floodThreshold))`